site/src/pages/RoadmapPage/Course.tsx

```diff
@@ -97,10 +97,12 @@ const Course: FC<CourseProps> = (props) => {
   const tappableCourseProps = props.addMode === 'tap' ? tapProps : {};
 
   return (
-    <div className={`course ${requiredCourses ? 'invalid' : ''}`} {...tappableCourseProps}>
+    <div className="course" {...tappableCourseProps}>
       <div className="course-card-top">
         <div className="course-and-info">
-          <CourseNameAndInfo data={props.data} {...{ openPopoverLeft, requiredCourses }} />
+          <span className={`${requiredCourses ? 'invalid-name' : ''}`}>
```

Comment:
`invalid-name` suggests that the user somehow entered the name and that it's wrong. I would keep the `invalid` className on the course itself but then change the style rule to be `.course.invalid .course-name` (or whatever the class name is for the element that contains the title and warning icon)